README.md

```diff
@@ -184,11 +184,11 @@ This plugin creates device plugin endpoints based on the configurations given in
         {
             "resourceName": "mlnx_sriov_rdma",
             "resourcePrefix": "mellanox.com",
-            "isRdma": true,
             "selectors": {
                 "vendors": ["15b3"],
                 "devices": ["1018"],
-                "drivers": ["mlx5_ib"]
+                "drivers": ["mlx5_ib"],
+                "isRdma": true,
```

Comment:
nit: comma is not needed, but it doesn't hurt.

Reply:
I will fix this. Thanks